tests/manifests/default/controller_definitions.yaml

```
@@ -248,141 +286,115 @@ controllerDefinitions:
-          - deletecollection
-          - get
-          - list
-          - patch
-          - update
-          - watch
-      - apiGroups:
-          - coordination.k8s.io
-        resources:
-          - leases
-        verbs:
-          - get
-          - list
-          - watch
-          - create
-          - update
-          - patch
-          - delete
-      - apiGroups:
-          - ""
-        resources:
-          - events
-        verbs:
-          - create
-          - patch
-      - apiGroups:
-          - ""
-        resources:
-          - pods
-          - pods/exec
-          - configmaps
-          - services
-          - persistentvolumeclaims
-        verbs:
-          - create
-          - get
-          - list
-          - watch
-          - update
-          - patch
-          - delete
-      - apiGroups:
-          - ""
-        resources:
-          - secrets
-        verbs:
-          - create
-          - get
-          - list
-          - update
-          - patch
-          - delete
-      - apiGroups:
-          - apps
-        resources:
-          - deployments
-          - statefulsets
-        verbs:
-          - create
-          - get
-          - list
-          - watch
-          - update
-          - patch
-          - delete
-      - apiGroups:
-          - batch
-        resources:
-          - jobs
-        verbs:
-          - create
-          - get
-          - list
-          - watch
-          - update
-          - patch
-          - delete
-    ---
-    apiVersion: rbac.authorization.k8s.io/v1
-    kind: RoleBinding
-    metadata:
-      labels:
-        app.kubernetes.io/component: controller-manager
-        app.kubernetes.io/name: numaflow-controller-manager
-        app.kubernetes.io/part-of: numaflow
-      name: numaflow-role-binding
-    roleRef:
-      apiGroup: rbac.authorization.k8s.io
-      kind: Role
-      name: numaflow-role
-    subjects:
-      - kind: ServiceAccount
-        name: numaflow-sa
-    ---
-    apiVersion: v1
-    kind: ServiceAccount
-    metadata:
-      name: numaflow-sa
-    ---
-    apiVersion: v1
-    data:
-      namespaced: "true"
-      server.disable.auth: "true"
-    kind: ConfigMap
+    apiVersion: apps/v1
+    kind: Deployment
     metadata:
-      name: numaflow-cmd-params-config
+      name: numaflow-controller
+    spec:
+      replicas: 1
+      selector:
+        matchLabels:
+          app.kubernetes.io/component: controller-manager
+          app.kubernetes.io/name: controller-manager
+          app.kubernetes.io/part-of: numaflow
+      template:
+        metadata:
+          labels:
+            app.kubernetes.io/component: controller-manager
+            app.kubernetes.io/name: controller-manager
+            app.kubernetes.io/part-of: numaflow
+        spec:
+          containers:
+          - args:
+            - controller
+            env:
+            - name: NUMAFLOW_IMAGE
+              value: quay.io/numaproj/numaflow:1.3.0-rc1
```

```suggestion
              value: quay.io/numaproj/numaflow:1.3.0
```

we have released 1.3.0

---

Do you want to replace that here @dpadhiar ?

---

Updated now to use image `v1.3.0`